app/models/food.rb

```diff
@@ -0,0 +1,9 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipes, through: :recipe_foods
+  has_many :recipe_foods
+
+  # def price=(value)
+  #   super(value&.delete('$'))
+  # end
```

Comment:
- [ ] Kindly remove this commented code to make the codebase look clean.
- [ ] Kindly add model validations for all your models.